src/index.js

```diff
@@ -112,28 +122,71 @@ class EHoldings extends Component {
         </Route>
       )
       : (
-        <CommandList commands={defaultKeyboardShortcuts}>
-          <KeyShortcutsWrapper focusSearchField={this.focusSearchField}>
-            <Route path={rootPath} component={ApplicationRoute}>
-              <Switch>
-                <Route path={`${rootPath}/providers/:providerId`} exact component={ProviderShow} />
-                <Route path={`${rootPath}/providers/:providerId/edit`} exact component={ProviderEdit} />
-                <Route path={`${rootPath}/packages/new`} exact component={PackageCreate} />
-                <Route path={`${rootPath}/packages/:packageId`} exact component={PackageShow} />
-                <Route path={`${rootPath}/packages/:packageId/edit`} exact component={PackageEdit} />
-                <Route path={`${rootPath}/titles/new`} exact component={TitleCreate} />
-                <Route path={`${rootPath}/titles/:titleId`} exact component={TitleShow} />
-                <Route path={`${rootPath}/titles/:titleId/edit`} exact component={TitleEdit} />
-                <Route path={`${rootPath}/resources/:id`} exact component={ResourceShow} />
-                <Route path={`${rootPath}/resources/:id/edit`} exact component={ResourceEdit} />
-                <Route path={`${rootPath}/notes/new`} exact component={NoteCreate} />
-                <Route path={`${rootPath}/notes/:noteId`} exact component={NoteView} />
-                <Route path={`${rootPath}/notes/:id/edit`} exact component={NoteEdit} />
-                <Route path={`${rootPath}/`} exact component={SearchRoute} />
-              </Switch>
-            </Route>
-          </KeyShortcutsWrapper>
-        </CommandList>
+        <>
+          <AppContextMenu>
+            {() => (
+              <NavList>
+                <NavListSection>
+                  <NavListItem
+                    id="eholdings-app-item"
+                    to="eholdings"
+                  >
+                    <FormattedMessage id="ui-eholdings.navigation.app" />
+                  </NavListItem>
+                  <NavListItem
+                    id="content-item"
+                    href="http://www.tfaforms.com/306425"
```

Comment:
There's a https version of this site. Let's ask Khalilah if we can use it instead of http

Reply:
Nice catch, fixed it